services/iam/src/routes/general.js

```diff
@@ -54,6 +54,49 @@ router.post('/login', authMiddleware.authenticate, authMiddleware.accountIsEnabl
 
 });
 
+router.post('/register', async (req, res, next) => {
+    const tenantName = req.body.companyName;
```

Comment:
You need to make sure your documentation matches the actual call.

API Doc: tenant
Route Body: companyName